tests/publiccloud/run_ltp.pm

```diff
@@ -141,18 +141,27 @@ sub run {
 
     assert_script_run($log_start_cmd);
 
-    my $cmd = 'perl -I runltp-ng runltp-ng/runltp-ng ';
-    $cmd .= '--logname=ltp_log --verbose ';
-    $cmd .= '--timeout=1200 ';
-    $cmd .= '--run ' . get_required_var('LTP_COMMAND_FILE') . ' ';
-    $cmd .= '--exclude \'' . get_var('LTP_COMMAND_EXCLUDE') . '\' ' if get_var('LTP_COMMAND_EXCLUDE');
-    $cmd .= '--backend=ssh';
-    $cmd .= ':user=' . $instance->username;
-    $cmd .= ':key_file=' . $instance->ssh_key;
-    $cmd .= ':host=' . $instance->public_ip;
-    $cmd .= ':reset_command=\'' . $reset_cmd . '\'';
-    $cmd .= ':ssh_opts=\'-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no\' ';
-    $cmd .= '--json_filter=openqa ';
+    zypper_call("in -y python3-paramiko python3-scp");
+
+    my $timeout=2400;
```

Comment:
1. can also try to fallback to initial 1200 as it was before ?
2. not sure if it make sense to keep it as variable . but if you think it is than we need to give it more concrete name ... e.g. "$ltp_suite_timeout"

Reply:
Actually this variable is to follow up discussion on it, to make update easy and also because it seems to depend on the sub-tests to run

Reply:
updated